{"name": "test_05_tourist_loginfail", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 < 0", "trace": "self = <test.test_LoginApi.TestLogin object at 0x000001CD0A853070>\n\n    @allure.feature(\"未完成模块\")\n    def test_05_tourist_loginfail(self):\n        #mylog.mylog.info('开始接口测试')\n>       assert 1 < 0\nE       assert 1 < 0\n\ntest221010\\test\\test_LoginApi.py:310: AssertionError"}, "start": 1670398200192, "stop": 1670398200192, "uuid": "b5cbd1ef-af85-484a-8fe9-37ba959b5a62", "historyId": "dc6e531ed4e8ec85a0615feb04566212", "testCaseId": "43f72ad0630a6da62b0cb5c8fd8ee6d8", "fullName": "test221010.test.test_LoginApi.TestLogin#test_05_tourist_loginfail", "labels": [{"name": "feature", "value": "未完成模块"}, {"name": "feature", "value": "登录注册模块"}, {"name": "parentSuite", "value": "test221010.test"}, {"name": "suite", "value": "test_LoginApi"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-8H78TQB"}, {"name": "thread", "value": "11216-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test221010.test.test_LoginApi"}]}